futures/src/subscription/tracker.rs

```diff
@@ -3,6 +3,50 @@ use crate::{BoxFuture, Subscription};
 use futures::{channel::mpsc, sink::Sink};
 use std::{collections::HashMap, marker::PhantomData};
 
+#[cfg(not(target_arch = "wasm32"))]
+mod trait_aliases {
+    use super::*;
+
+    pub trait TrackerMessage: Send + 'static {}
+
+    impl<T> TrackerMessage for T where T: Send + 'static {}
+
+    pub trait TrackerMessageReceiver<Message: TrackerMessage>:
+        Sink<Message, Error = mpsc::SendError> + Unpin + Send + Clone + 'static
+    {
+    }
+
+    impl<Message: TrackerMessage, T> TrackerMessageReceiver<Message> for T where
+        T: Sink<Message, Error = mpsc::SendError>
+            + Unpin
+            + Send
+            + Clone
+            + 'static
+    {
+    }
+}
+
+#[cfg(target_arch = "wasm32")]
+mod trait_aliases {
+    use super::*;
+
+    pub trait TrackerMessage: 'static {}
+
+    impl<T> TrackerMessage for T where T: 'static {}
+
+    pub trait TrackerMessageReceiver<Message: TrackerMessage>:
+        Sink<Message, Error = mpsc::SendError> + Unpin + Clone + 'static
+    {
+    }
+
+    impl<Message: TrackerMessage, T> TrackerMessageReceiver<Message> for T where
+        T: Sink<Message, Error = mpsc::SendError> + Unpin + Clone + 'static
+    {
+    }
+}
+
+pub use trait_aliases::{TrackerMessage, TrackerMessageReceiver};
```

Comment:
Sadly yes, this is necessary - we use `winit` here with WebGL backend and it (winit) does not implement `Send` on its `EventLoopProxy` on wsam32, so yea, because of that we should lift here requirement for `Send` for our types too